### PR TITLE
Explain how to solve Fastlane is slow in th FAQ

### PR DESCRIPTION
**Pull request by lacostej**
This is related to

https://github.com/fastlane/fastlane/issues/420
https://github.com/fastlane/fastlane/issues/1293
https://github.com/fastlane/fastlane/issues/1417

### CODE REVIEW

**Comment by KrauseFx** [comment]
We also have it covered here https://github.com/fastlane/fastlane#installation, but we can't have enough of those warnings :+1:

File: fastlane/docs/FAQs.md [hunk 1]
```diff
@@ -21,6 +21,24 @@ If you use `rvm`, try the following
 rvm osx-ssl-certs update all
 ```
 
+### Fastlane is slow (to start)
+
+Fastlane uses rubygems and [rubygems is known to be slow itself](http://www.sitepoint.com/rubygems-slow/)
```
Comment (KrauseFx) [inline fastlane/docs/FAQs.md]: That's unnecessary harsh...

**Comment by lacostej** [comment]
@KrauseFx I didn't see it and I only saw `gem cleanup` recommended. It's missing the `bundle exec` solution which IMHO is better in many cases.

File: fastlane/docs/FAQs.md (hunk 1 above)
Comment (lacostej) [inline fastlane/docs/FAQs.md]: I didn't mean it as harsh. I found that particular article very enlightening :) Should I change the wording ?

File: fastlane/docs/FAQs.md [hunk 2]
```diff
@@ -21,6 +21,24 @@ If you use `rvm`, try the following
...
+
+If you see abnormal slowdown when you run fastlane, here are 2 known ways to reduce this problem:
+
+* if you use fatlane with a Gemfile, use `bundle exec` to reduce the amount of searched gems. E.g.
```
Comment (ffittschen) [inline fastlane/docs/FAQs.md]: @lacostej `fatlane` pun intended? :smile:

Comment (lacostej) [inline fastlane/docs/FAQs.md]: can't even hide Easter Eggs in the docs without all those seeing eyes anymore :)

:clap:

Comment (KrauseFx) [inline fastlane/docs/FAQs.md]: So mean :stuck_out_tongue_winking_eye:

File: fastlane/docs/FAQs.md (hunk 1 above)
Comment (KrauseFx) [inline fastlane/docs/FAQs.md]: Let's just remove the whole line and stick with the rest

File: fastlane/docs/FAQs.md [hunk 3]
```diff
@@ -21,6 +21,24 @@ If you use `rvm`, try the following
...
+
+```
+bundle exec fastlane actions
```
Comment (KrauseFx) [inline fastlane/docs/FAQs.md]: We can also link to https://guides.cocoapods.org/using/a-gemfile.html for more information

**Comment by mfurtak** [comment]
Hi folks - where do we stand on this?

**Comment by jeffreyjackson** [comment]
ive solved this issue before by also doing a `sudo gem cleanup`

**Comment by KrauseFx** [comment]
Moved over to https://github.com/fastlane/fastlane/pull/3893 :+1:

**Pull request closed without merging**